BUILD.md

```diff
@@ -9,17 +9,16 @@ Microsoft Windows
 
 ### Prerequisites
 
-1. Install free [Microsoft Visual Studio] Community 2017. Select following
+1. Install free [Microsoft Visual Studio] Community 2017. Select at least the following
```

Comment:
This can be risky as additional problems may arise if multiple toolsets or SDKs are installed at the same time